### PR TITLE
refactor: Clean warning and messages for project ArmoniK.DevelopmentKit.Client.Common

### PR DESCRIPTION
**Pull request by wkirschenmann**

### CODE REVIEW

File: Tests/ArmoniK.EndToEndTests/ArmoniK.EndToEndTests.Client/Tests/AggregationPriority/AggregationPriorityTest.cs [hunk 1]
```diff
@@ -150,23 +150,49 @@ public void Check_That_serialazation_is_ok()
...
 
-    var taskRawData = new List<TaskRaw>();
+    var taskRawData = new List<TaskDetailed>();
 
     await foreach (var taskRaw in RetrieveAllTasksStats(service.GetChannel(),
-                                                        new ListTasksRequest.Types.Filter
+                                                        new Filters
                                                         {
-                                                          SessionId = service.SessionId,
+                                                          Or =
```
Comment (aneojgurhem) [inline Tests/ArmoniK.EndToEndTests/ArmoniK.EndToEndTests.Client/Tests/AggregationPriority/AggregationPriorityTest.cs]: This should not appear in this PR as it was done in another one

File: Client/src/Common/Status/ArmonikStatusCode.cs [hunk 2]
```diff
@@ -57,7 +63,8 @@ public enum ArmonikStatusCode
   ResultNotReady,
 
   /// <summary>
-  ///   The result is in error and the task could finished without no result
+  ///   The result is in error and the task could finished with no result
```
Comment (aneojgurhem) [inline Client/src/Common/Status/ArmonikStatusCode.cs:66]:
```suggestion
  ///   The result is in error and the task could finished without result
```

File: Tests/ArmoniK.EndToEndTests/ArmoniK.EndToEndTests.Client/ArmoniK.EndToEndTests.Client.csproj [hunk 3]
```diff
@@ -26,6 +26,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="ArmoniK.Api.Common" Version="3.11.0" />
```
Comment (aneojgurhem) [inline Tests/ArmoniK.EndToEndTests/ArmoniK.EndToEndTests.Client/ArmoniK.EndToEndTests.Client.csproj:29]: is this really needed ?

File: Client/src/Common/Submitter/BaseClientSubmitter.cs [hunk 4]
```diff
@@ -819,32 +817,33 @@ public ICollection<GetResultIdsResponse.Types.MapTaskResult> GetResultIds(IEnume
...
-
-        throw new ClientResultsException(msg,
-                                         resultStatus.IdsError.ToArray());
+                              ? resultStatus.IdsError[0]
+                              : resultStatus.IdsResultError[0].TaskId;
+
+        Logger.LogError("The missing result is in error or canceled. Please check log for more information on Armonik grid server list of taskIds in Error: [{taskList}]",
+                        ids);
+        Logger.LogError("1st result id where the task which should create it is in error : {taskIdInError}",
+                        taskIdInError);
```
Comment (aneojgurhem) [inline Client/src/Common/Submitter/BaseClientSubmitter.cs:841]: There two logs are redundant with the following error. There will have 3 logs in error with the same information

Comment (wkirschenmann) [reply Client/src/Common/Submitter/BaseClientSubmitter.cs:841]: The two log are not the same. I'll remerge them.

**Review by aneojgurhem — APPROVED** [review]

**Review by aneojgurhem — APPROVED** [review]

**Pull request merged**